addons/hr/models/res_config_settings.py

```diff
@@ -22,6 +20,4 @@ class ResConfigSettings(models.TransientModel):
 
     @api.constrains('module_hr_presence', 'hr_presence_control_email', 'hr_presence_control_ip')
     def _check_advanced_presence(self):
-        test_mode = self.env.registry.in_test_mode() or getattr(threading.current_thread(), 'testing', False)
-        if self.env.context.get('install_mode', False) or test_mode:
-            return
+        pass
```

Comment:
I guess this won't stay like that right ?

Reply:
Removed altogether.